tutorial/create-test-virtual-device.md

```diff
@@ -0,0 +1,212 @@
+
+## Create the device
+
+The first step is to create a `yaml` file that can be used as the application manifest. It is important that this file is created in your `home` directory.
+
+In the home directory, let's first create a directory for the application content:
+
+      mkdir my-app
+
+This directory is the location where the application manifest file and any relevant APK must reside.
+
+For this tutorial, we are going to create a device with a basic application without an APK that will start directly with the Android system launcher. Let's create the `manifest.yaml` inside the `my-app` directory with the following contents:
+
+```yaml
+name: my-first-app
+resources:
+   cpus: 4
+   memory: 4GB
+   disk-size: 3GB
+```
+
+Now, let's create the application:
+
+      amc application create my-app/
+
+`my-app/` is the application directory where the manifest and any other application related artifacts are available.
+
+The application is created and bootstrapped. Now, to see the application details and its status, run:
+
+      amc application list
+
+Watch the output of this command till the application becomes *ready*.
+
+## Test the virtual device
+
+**Launch an instance**
+
+When the application is *ready*, let's launch an instance:
+
+      amc launch my-first-app --vm --enable-streaming
+
+This command creates and starts an instance for the application.
+
+`my-first-app` is the application name, you can also use the application ID to launch an instance for it.
+
+The `--enable-streaming` flag indicates that we want to stream this instance.
+
+The `--vm` flag indicates that we want to create the application inside a virtual machine instance instead of the default container instance.
+
+To see the instance details and its status, run:
+
+      amc ls
+
+When the instance status reaches the *running* status, let's try to access it. We will need to copy the instance ID from the previous command's output.
+
+      amc shell instance_id
+
+This takes us into the Linux instance that runs the Android container. Next, let's try accessing the nested Android container:
+
+      anbox-shell
+
+Now, we are inside the Android container. Play around by running some of the following commands:
+
+      ls
+      logcat
+      exit
+
+`exit` or pressing Control+D takes us back to the Linux container.
+
+**Update an application**
+
+We can have several versions of an application. Let's try and update the application:
+
+Go to the application directory and create a new manifest. This manifest introduces a change to the disk-size and also has a tag to associate with the application.
+
+```yaml
+name: my-first-app
+resources:
+   cpus: 4
+   memory: 4GB
+   disk-size: 8GB
+tags:
+   game
+```
+
+Now, run the following commands and observe the output:
+
+      amc application update my-first-app my-app/
+      amc application list
+      amc application show my-first-app
```

Comment:
Same above, it'd be much better if we could include the output of the above commands